layouts/partials/analytics.html

```diff
@@ -22,7 +22,10 @@
                     screen: screen.width + "x" + screen.height,
                     user: "{{ .id }}",
                     utcoffset: "1",
-                    })
+                    }),
+		    {{ if .noReferrerWhenDowngrade }}
```

Comment:
It will be better to use a variable `referrerPolicy` instead of `noReffererWhenDowngrade`. Then, user will have option to use other policies too.

We can just do:
```
{{ if .referrerPolicy }}
  { referrerPolicy: "{{ .referrerPolicy}}" }
{{ end }}
```

Reply:
I believe this is the correct form for such a task. Is there anything else that I can do (or something that I have done wrong)?

Reply:
Thank you. Now, it looks good to me.